java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -322,6 +324,29 @@ public void set_only_if_does_not_exists_missing_key(BaseClient client) {
         assertEquals(ANOTHER_VALUE, data);
     }
 
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void set_get_binary_data(BaseClient client) {
+        GlideString key = gs("set_get_binary_data_key");
+        byte[] binvalue = {(byte) 0x01, (byte) 0x00, (byte) 0x01, (byte) 0x00, (byte) 0x02};
+        assert client.set(key, gs(binvalue)).get().equals("OK");
+        GlideString data = client.get(key).get();
+        assert Arrays.equals(data.getBytes(), binvalue);
```

Comment:
`data.equals(gs(binvalue))` should work too
With the correct assert it will be
```suggestion
        assertEquals(data, gs(binvalue));
```
or
```suggestion
        assertArrayEquals(data.getBytes(), binvalue);
```